docs/src/main/asciidoc/security-openid-connect-client.adoc

```diff
@@ -3,18 +3,18 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= OpenID Connect Client and Token Propagation Quickstart
+= OpenID Connect client and token propagation quickstart
 include::_attributes.adoc[]
 :categories: security
-:summary: This guide explains how to use OpenID Connect and OAuth2 Client and Filters to acquire, refresh and propagate access tokens.
+:summary: This guide explains how to use OpenID Connect and OAuth2 Client and Filters to acquire, refresh, and propagate access tokens.
 :topics: security,oidc,client
 :extensions: io.quarkus:quarkus-oidc-client
 
-This quickstart demonstrates how to use `OpenID Connect Client Reactive Filter` to acquire and propagate access tokens as `HTTP Authorization Bearer` access tokens, alongside `OpenID Token Propagation Reactive Filter` which propagates the incoming `HTTP Authorization Bearer` access tokens.
+This quickstart demonstrates how to use `OpenID Connect Client Reactive Filter` to acquire and propagate access tokens as `HTTP Authorization Bearer` access tokens alongside `OpenID Token Propagation Reactive Filter`, which propagates the incoming `HTTP Authorization Bearer` access tokens.
 
-Please check xref:security-openid-connect-client-reference.adoc[OpenID Connect Client and Token Propagation Reference Guide] for all the information related to `Oidc Client` and `Token Propagation` support in Quarkus.
+For more information about `Oidc Client` and `Token Propagation` support in Quarkus, see the xref:security-openid-connect-client-reference.adoc[OpenID Connect (OIDC) and OAuth2 Client and Filters Reference Guide.
 
-Please also read xref:security-oidc-bearer-token-authentication.adoc[OIDC Bearer token authentication] guide if you need to protect your applications using Bearer Token Authorization.
+To protect your applications by using Bearer Token Authorization, see the xref:security-oidc-bearer-token-authentication.adoc[OpenID Connect (OIDC) bearer token authentication] guide.
```

Comment:
@sheilamjones I'd keep the `authentication`, for now at least, as we use this term alongside the bearer token in different places, since in context of Quarkus, and bearer access tokens specifically, even though, in a pure original OAuth2 sense, this token represents a user authorizing the client to access some service on this user's behalf, technically, in Quarkus, it is an authentication mechanism that has to verify this token.

May be can have a separate discussion/PR about more precise messages related to bearer tokens, if it is the authentication or authorization, etc

Reply:
Left it mostly as is, pending a terminology conversation. Updated the capitalization of the link text.

Reply:
Thanks @rolfedh. Thanks @sberyozkin. Yes, I think we perhaps need a separate discussion to clarify the usage of "authorization" versus "authentication" in the context of the bearer access tokens. 
With regards to your comment @sberyozkin, you suggested to keep "**authentication"** in the above sentence 17, however it stated "Bearer Token **Authorization**" in the original sentence (and also our the updated restructured proposal). Therefore, @rolfedh think this should therefore be changed to "To protect your applications by using Bearer **token authentication**, see...."

Reply:
@sheilamjones let's handle that terminology fix separately, across multiple files, like I did with Fix terminology: Replace configuration key with configuration property #37591

Reply:
I have created https://github.com/quarkusio/quarkus/issues/37698 for this issue.

Reply:
Thanks @rolfedh. Shall therefore approve this PR pending the follow-up conversation in PR https://github.com/quarkusio/quarkus/issues/37698.